sdk/storage/storage-blob/src/AccountSASSignatureValues.ts

```diff
@@ -118,6 +118,30 @@ export function generateAccountSASQueryParameters(
     ? accountSASSignatureValues.version
     : SERVICE_VERSION;
 
+  if (
+    accountSASSignatureValues.permissions &&
+    accountSASSignatureValues.permissions.deleteVersion &&
+    version < "2019-10-10"
+  ) {
+    throw RangeError("'version' must be >= '2019-12-12' when provided 'x' permission.");
+  }
```

Comment:
Fixed in https://github.com/ljian3377/azure-sdk-for-js/commit/78cbb6c1fe5226cd47c07018828f5ab47059b5bd
```
throw RangeError("'version' must be >= '2019-10-10' when provided 'x' permission.");
```